src/pages/Checkout/CheckoutPage.vue

```diff
@@ -528,6 +529,12 @@ export default {
 		this.getUpsellModuleData();
 	},
 	computed: {
+		isUpsellUnder100() {
+			const amountLeft = this.upsellLoan?.loanAmount
+			- this.upsellLoan?.loanFundraisingInfo?.fundedAmount
+			- this.upsellLoan?.loanFundraisingInfo?.reservedAmount;
```

Comment:
Fallback numbers (like 0) will be needed if these props are indeed ever undefined. There's optional chaining here, but I'm not sure if they are ever undefined in this situation. A number - undefined = NaN.

Reply:
yeah this seems to just be the standard across the ui repo for `amountLeft` but added a `0` fallback 👍